packages/suite/src/views/wallet/staking/components/EthStakingDashboard/components/EmptyStakingCard.tsx

```diff
@@ -69,86 +41,83 @@ export const EmptyStakingCard = () => {
         () => [
             {
                 id: 0,
-                icon: <Icon name="piggyBank" size={32} color={theme.iconPrimaryDefault} />,
+                icon: <Icon name="piggyBank" size="extraLarge" variant="primary" />,
                 title: <Translation id="TR_STAKE_ETH_SEE_MONEY_DANCE" />,
                 description: (
                     <Translation
                         id="TR_STAKE_ETH_SEE_MONEY_DANCE_DESC"
                         values={{
                             apyPercent: ethApy,
+                            t: text => (
+                                <Tooltip
+                                    dashed
+                                    isInline
+                                    content={<Translation id="TR_STAKE_APY_DESC" />}
+                                >
+                                    <abbr>{text}</abbr>
```

Comment:
I this there should be two components: 
- Tooltip (simple tooltip)
- Abbr (has icon, can have underline, uses Tooltip internally, semantically abbr tag)
 
WDYT?

If you agree and you want to refactor it, you can do it in another PR to not block this one.

Reply:
I agree with the underline moving out of Tooltip and into a new Abbr component. But I think the Tooltip should keep the hasIcon prop and the Abbr should use the Tooltip internally (with no icon) with a simple title prop.

Reply:
Not sure, but we can discuss about it later